index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,900;1,100;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Harmattan&family=Roboto:ital,wght@0,900;1,100;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <title>Gideon's Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/9947fbd823.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+
+  <body>
+    <header class="template5-main-mobile">
+      <div class="app-bar">
+        <div class="time"><img src="Time.png" alt="time" /></div>
+        <a href="#" class="name">Gidiz Logo</a>
+      </div>
+      <i class="fa-solid fa-bars"></i>
+      <div class="ellipse1"></div>
+      <div class="ellipse2"></div>
+
+      <section class="headline">
+        <h1 class="primary-text">Gideon Dadi</h1>
+        <h3 class="supporting-text">Software Engineer</h3>
+
+        <div class="advert-text">
+          <p>
+            I can help you build a product, feature or website Look through some
+            of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <hr />
+      </section>
+      <section class="section-2">
+        <div class="language">
+          <h2>Language</h2>
+          <p>Javascript</p>
+          <p>Ruby</p>
+          <p>Node</p>
```

Comment:
- [ ] Please, consider that you have to use two sections: `header` and `section`. Here you have added an additional `section` tag with class name `section-2` - you should use for all headline section only one `section` tag. So, just put everything in one `<section>` ... `</section>`

Reply:
Noted